src/interfaces/mcp-options.interface.ts

```diff
@@ -4,7 +4,9 @@ import { CanActivate } from '@nestjs/common';
 export enum McpTransportType {
   SSE = 'sse',
   STREAMABLE_HTTP = 'streamable-http',
+  // TODO: change name since now we have stdio
```

Comment:
Which name we can put here? `ALL_HTTP` maybe?

Reply:
Maybe a better UX is to start all 3 types and have options to disable those?